.env.example

```diff
@@ -1 +1,3 @@
-PORT=
\ No newline at end of file
+PORT=
+
+POSTGRES_URL=
```

Comment:
Uma dica para aplicações que vão ser feitos o deploy no heroku, o padrão é ter uma variável chamada "DATABASE_URL", o heroku gerencia ela e caso ele suba outra banco ele se encarrega de atualizar ela. Eu prefiro usar esse nome pq é o padrão no mercado. Se tu for fazer deploy no heroku e a URL do banco mudar tu tem que ir lá e atualizar manualmente.